include/boost/buffers/buffer_copy.hpp

```diff
@@ -86,10 +86,11 @@ struct buffer_copy_impl
                     n = b1.size();
                 if( n > at_most - total)
                     n = at_most - total;
-                std::memcpy(
-                    b1.data(),
-                    b0.data(),
-                    n);
+                if(n != 0)
```

Comment:
I think our style guide would have this written as `if( n != 0 )`

Reply:
`if(n != 0)` conforms to the style of other if statements in this repository. should I change it to `if( n != 0 )`?